themes/bootstrap5/js/common.js

```diff
@@ -293,46 +348,35 @@ var VuFind = (function VuFind() {
    * @param {string}  property Target property ('innerHTML', 'outerHTML' or '' for no HTML update)
    */
   function setElementContents(elm, html, attrs = {}, property = 'innerHTML') {
-    // Extract any scripts from the HTML and add them separately so that they are executed properly:
+    const tmpDiv = el('div', '', {}, stringToNodes(html));
```

Comment:
It feels a little weird that we use `stringToNodes` here to create a div and pull out its contents, and then we use `el` to create a _different_ div and put those elements back inside... and then down below, we pull the elements back out of the second div on various code paths. Would it be better to refactor this to just use the result of `stringToNodes` directly without creating a second container? I realize this would prevent `querySelectorAll` from being used below, but is there perhaps some other way of filtering the array to achieve the same effect? Or, since `stringToNodes` is only used in one place, should we consider defining it differently or giving it an argument to just avoid stripping the container in the first place?

Reply:
hmm, could try and have a look!

Reply:
Hmm, backtracked a littlebit here, as stringToNodes was used in only this place and added the assignment to innerhtml here. Idea is that there is one path to use and if any sanitization is required then its easier to manage the content when its all set with the same function.